assets/constants.ts

```diff
@@ -4847,6 +4847,10 @@ export default {
     "code": "lt",
     "description": "lietuvių (Lithuanian)",
     "direction": "ltr"
+  }, {
+    "code": "ms",
+    "description": "Bahasa Melayu (Malay)",
```

Comment:
Ignore this comment, I found the local term is `Bahasa Melayu`.

Reply:
I wanted to make sure that we are expecting this to be the content language and not a voiceover/translation language, right?

Reply:
I found the reason for this change in #12341, I've left a reply in the issue thread!

Reply:
Sorry for the misunderstanding, let me rephrase it:

Currently, the dropdown list is not showing the langaugeDescription because we are using [getContentLanguageDescription](https://github.com/oppia/oppia/blob/ed8c39519cb18854eb61bc971d47d2aa89eb904e/core/templates/pages/exploration-player-page/services/content-translation-language.service.ts#L93) instead of [getAudioLanguageDescription](https://github.com/oppia/oppia/blob/ed8c39519cb18854eb61bc971d47d2aa89eb904e/core/templates/domain/utilities/language-util.service.ts#L155).

If we go with the existing fix in this PR then it will fix the issue for one specific language, not for other languages that don't have mentions in contentLanaguage list.

Reply:
Yup, I understand. The "fix" in this PR is a band-aid and is not meant to be a general thing. However, it does make things "more correct", since there's no reason why Malay shouldn't be supported in both forms. A further fix is needed to handle audio properly, though, and this isn't a replacement for that further fix.

Reply:
Assume someone wrote translation in "Dari" language which only exists in audioLanaguge and not in contentLanaguage, then the learners won't be able to see the language description, right? (i.e, the learner will still face this issue)

I think a "band-aid and minimal" fix would be to make it work in the existing bad structure (existing bad structure = only the languages in supported audio will have translations)

Reply:
As I have mentioned above, this PR is not meant to fix #12341. I only included it because it fixes an error I saw when trying to test the functionality locally.

If you like, I'm happy to drop the fix for Malay from this PR.

Reply:
I'm fine with either of these:
1. Change getContentLanguageDescription to getAudioLanguageDescription [here](https://github.com/oppia/oppia/blob/ed8c39519cb18854eb61bc971d47d2aa89eb904e/core/templates/pages/exploration-player-page/services/content-translation-language.service.ts#L93).
2. Adding Dari and Malay in SUPPORTED_CONTENT_LANGUAGES list.
3. Fix #12341 in a separate PR.

If going for 2 or 3 then #12341 should be fixed quickly (before the next release cut) [As the 1st bullet will fix the issue completely for the existing structure.]

Reply:
Update: We discussed this offline, and decided that in this PR, we'll do (1). This is because "content language" is actually the list of languages that an exploration can be created in, and does not actually relate to the languages that translations/voiceovers cover, so it is a less good choice than "audio language" although the latter is not, strictly speaking, correct. The further long-term fix will be deferred to #12341. I'll also drop Malay in the SUPPORTED_CONTENT_LANGUAGES since it is not relevant to this PR.

Reply:
Further update: it turns out that this has unintended consequences because it changes the dropdown label from e.g. "中文 (Chinese)" to "Chinese", which is a regression.

Per discussion with @DubeySandeep, leaving it for now, and deferring a fuller fix to #12341.